onnx/defs/tensor/defs.cc

```diff
@@ -317,6 +317,10 @@ ONNX_OPERATOR_SET_SCHEMA(
               !hasNInputShapes(ctx, static_cast<int>(numInputs))) {
             return;
           }
+          if (numInputs == 1) {
+            propagateShapeFromInputToOutput(ctx, 0, 0);
```

Comment:
Would be better to move this down so it's after the axis is validated.

Reply:
Sorry for the latency, done!